### PR TITLE
fix: #673 Application not save empty 'About' field on user page

### PR DESCRIPTION
**Pull request by Moon1it**
https://github.com/tritonJS826/masters-way/issues/673

### CODE REVIEW

File: mw-webapp/src/component/select/Select.cy.tsx [hunk 1]
```diff
@@ -65,16 +62,18 @@ describe("Select component", () => {
...
   it("should change value inside trigger if some selectItem selected", () => {
-    cy.get(getDataCy(SELECT_CY.dataCyValue)).contains(SELECT_OPTIONS[FIRST_OPTION_INDEX].text);
+    cy.get(getDataCy(SELECT_CY.dataCyValue)).contains(
+      SELECT_OPTIONS[FIRST_OPTION_INDEX].text,
+    );
     cy.get(getDataCy(SELECT_CY.dataCyTrigger)).click();
-    cy.get(getDataCy(SELECT_CY.dataCyContentList)).click();
-    cy.get(getDataCy(SELECT_CY.dataCyValue)).contains(SELECT_OPTIONS[SECOND_OPTION_INDEX].text);
+    // Force required for correct work for MacOS
+    cy.get(getDataCy(SELECT_CY.dataCyContentList)).click({force: true});
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/select/Select.cy.tsx:70]: Can you please tell more about why on MacOs doesn't work click()? Or send link please on resource

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/select/Select.cy.tsx (hunk 1 above)
Comment (Moon1it) [reply mw-webapp/src/component/select/Select.cy.tsx:70]: I received this error!
```
Select component
should change value inside trigger if some selectItem selected:
AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Select text 2' within the element: ‹span> but never did.
```

**Review by tritonJS826 — APPROVED** [review]

**Review by Ekaterina1994 — APPROVED** [review]

File: mw-webapp/src/component/select/Select.cy.tsx [hunk 2]
```diff
@@ -65,16 +62,18 @@ describe("Select component", () => {
   });
 
   it("should change value inside trigger if some selectItem selected", () => {
-    cy.get(getDataCy(SELECT_CY.dataCyValue)).contains(SELECT_OPTIONS[FIRST_OPTION_INDEX].text);
+    cy.get(getDataCy(SELECT_CY.dataCyValue)).contains(
+      SELECT_OPTIONS[FIRST_OPTION_INDEX].text,
```
[the remark this answers is not in the document]
Comment (Moon1it) [reply mw-webapp/src/component/select/Select.cy.tsx]: Done

File: mw-webapp/src/component/select/Select.cy.tsx [hunk 3]
```diff
@@ -22,10 +22,10 @@ const SELECT_OPTIONS = [
...
 
-  /**
-   * SelectTest onChange
-   */
-  onChange: (value: string) => void;
+	/**
+	 * SelectTest onChange
+	 */
+	onChange: (value: string) => void;
 }
```
[the remark this answers is not in the document]
Comment (Moon1it) [reply mw-webapp/src/component/select/Select.cy.tsx]: Done

**Pull request merged**